openedx-devstack-ubuntu/azuredeploy.json

```diff
@@ -0,0 +1,262 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "newStorageAccountName": {
+      "type": "string",
+      "metadata": {
+        "description": "The name of the new storage account created to store the VMs disks. Name must be between 3 and 24 characters and use lowercase alphanumeric only."
+      }
+    },
+    "adminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Administrator password. Username is 'vagrant'."
+      }
+    },
+    "dnsNameForPublicIP": {
+      "type": "string",
+      "metadata": {
+        "description": "Unique DNS name for the Public IP used to access the Virtual Machine."
+      }
+    },
+    "vmSize": {
+      "type": "string",
+      "defaultValue": "Standard_D2_v2",
+      "allowedValues": [
+        "Standard_A1",
+        "Standard_A2",
+        "Standard_A3",
+        "Standard_A4",
+        "Standard_D1",
+        "Standard_D2",
+        "Standard_D3",
+        "Standard_D4",
+        "Standard_D1_v2",
+        "Standard_D2_v2",
+        "Standard_D3_v2",
+        "Standard_D4_v2"
+      ],
+      "metadata": {
+        "description": "Virtual machine size."
+      }
+    }
+  },
+  "variables": {
+    "apiVersion": "2015-06-15",
+    "openedxVersion": "named-release/dogwood.1",
+    "scriptDownloadUri": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/openedx-devstack-ubuntu/master/",
+    "installScript": "install-openedx.sh",
+    "installCommand": "[concat('sudo bash -c ''nohup ./', variables('installScript'), ' ', variables('openedxVersion'), ' </dev/null &>/var/log/azure/openedx-install.log &''')]",
+    "adminUsername": "vagrant",
+    "vmName": "jumpbox2",
+    "osImagePublisher": "Canonical",
+    "osImageOffer": "UbuntuServer",
+    "osImageSKU" : "12.04.5-LTS",
```

Comment:
Open edX installation docs specifically call out 12: https://openedx.atlassian.net/wiki/display/OpenOPS/Open+edX+Installation+Options

Saw in forums that some users are on 14, but I'm hoping to do this in a future commit once I've done more testing. I do realize Azure support for 12 ends on 4/17:
https://azure.microsoft.com/en-us/marketplace/partners/canonical/ubuntuserver1204lts/